core/domain/question_services.py

```diff
@@ -217,3 +223,254 @@ def update_question(
     updated_question = apply_change_list(question_id, change_list)
     _save_question(
```

Comment:
Increment version here. Version is not the universal number for all objects as I told before. It increments as edits are done to a question model.

Reply:
Okay done.